src/app/domain/geolocation/tracking/index.js

```diff
@@ -74,6 +84,26 @@ export const stopTracking = async () => {
   }
 }
 
+export const getTrackingConfig = async () => {
+  const localTrackingConfig = await getData(
+    StorageKeys.GeolocationTrackingConfig
+  )
+
+  return localTrackingConfig ?? DEFAULT_TRACKING_CONFIG
```

Comment:
I didn't know this surprising syntax `??` , I assume this is a shorthand for `localTrackingConfig ? localTrackingConfig : DEFAULT_TRACKING_CONFIG` ?

Reply:
Exactly https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing